### PR TITLE
updating FAQ, adding coin allocation Q/A

### PR DESCRIPTION
**Pull request by b00f**
This PR adds a new FAQ for initial token allocation.

### CODE REVIEW

File: website/_i18n/en.yml [hunk 1]
```diff
@@ -140,7 +140,7 @@ dict:
     title: FAQ
     general: General
     validator: Validator
-    economy: Economy
+    economy: Tokenomy
```
Comment (Mr-HoDL58) [inline website/_i18n/en.yml:143]: Not sure this is the right description. Allocation or economy?

File: website/_i18n/en.yml [hunk 2]
```diff
@@ -312,10 +311,58 @@ dict:
       especially when they want to stake more than the maximum stake per validator.
       This eliminates the need for additional resources and simplifies node management.
     staking_limit: Why is there a limit for maximum stake?
-    staking_limit_answer_v1:
+    staking_limit_answer_v1: >
       The limit for the maximum stake per validator enhances decentralization and prevents validators from gaining too much power,
       which could potentially lead them to control the committee.
       The maximum staking limit is part of the consensus parameters and can be changed in the future through consensus among the validators.
+    genesis_allocation: How are Genesis Tokens allocated in Pactus?
```
Comment (Mr-HoDL58) [inline website/_i18n/en.yml:318]: Word token is used,

File: website/_i18n/en.yml [hunk 3]
```diff
@@ -312,10 +311,58 @@ dict:
...
+    genesis_allocation_answer_v1: >
+      The initial allocation of Genesis Tokens in the Pactus blockchain is distributed as follows:
```
Comment (Mr-HoDL58) [inline website/_i18n/en.yml:320]: Word token is used.

File: website/_i18n/en.yml [hunk 4]
```diff
@@ -312,10 +311,58 @@ dict:
...
+      <br><br>
+      <table border="1">
+        <caption>Genesis Token Allocation for Pactus</caption>
```
Comment (Mr-HoDL58) [inline website/_i18n/en.yml:323]: Word token is used.

File: website/_i18n/en.yml [hunk 5]
```diff
@@ -312,10 +311,58 @@ dict:
...
+        <thead>
+            <tr>
+                <th>Category</th>
+                <th>Token Allocation</th>
```
Comment (Mr-HoDL58) [inline website/_i18n/en.yml:327]: Word token is used.

File: website/_i18n/zh.yml [hunk 6]
```diff
@@ -130,7 +130,7 @@ dict:
     title: 常见问题解答
     general: 常规问题
     validator: 验证节点问题
-    economy: 经济问题
+    economy: Tokenomy
```
Comment (Mr-HoDL58) [inline website/_i18n/zh.yml:133]: Not sure this is the right description. Allocation or economy?

File: website/learn/blockchain/genesis/index.md [hunk 7]
```diff
@@ -40,8 +40,9 @@ These parameters are hardcoded into the project and include:
   are defined at genesis time and ensure that the entire network operates within the same configuration.
   These consensus parameters are discussed in detail in the consensus section of the documentation.
 - **Treasury Account**: The [treasury account]({{ site.baseurl }}/learn/blockchain/account/#treasury-account)
-  is defined at the genesis time and holds the total supply of the Pactus blockchain,
-  21 million coins. Each coin is divided into 1 billion units.
+  holds  21 million coins at the genesis time. Each coin is divided into 1 billion units.
+- **Reserved Accounts**: Reserved account are defined at the Genesis time and at total they have 21 milion coins.
+  These coins are reserved and the main purpise is to support the project over time.
```
Comment (Mr-HoDL58) [inline website/learn/blockchain/genesis/index.md:45]: We should add a decription of the type of security for the reserve accounts.

Comment (Mr-HoDL58) [reply website/learn/blockchain/genesis/index.md:45]: no doubt we will be asked questions if we dont put anything.

**Comment by Mr-HoDL58** [comment]
ttles and text should be free of the word token. Coin/s or Native Coin/s

File: website/learn/blockchain/genesis/index.md (hunk 7 above)
Comment (b00f) [reply website/learn/blockchain/genesis/index.md:45]: Agreed! The best approach is to define a threshold signature address for these accounts (or, as YK once suggested, a vesting account).
A vesting account is an interesting idea; we can schedule the release of assets in the future.
However, the destination address is still a question mark.

We can even think about validators get approve for the signature (like 50% of total stake). Then here validators get too much influence on the network. 

At the moment, I think threshold signature is the best solution. Hopefully, we can implement it in the core.

**Pull request merged**